man/man8/zpool-scrub.8

```diff
@@ -92,9 +93,27 @@ Once resumed the scrub will pick up from the place where it was last
 checkpointed to disk.
 To resume a paused scrub issue
 .Nm zpool Cm scrub
+or
+.Nm zpool Cm scrub
+.Fl e
 again.
 .It Fl w
 Wait until scrub has completed before returning.
+.It Fl e
+Only scrub blocks in the error log.
+This is only supported if the head_errlog feature is enabled in the pool.
+It is strongly advised to run error scrubs with keys loaded and filesystems
+mounted in the case of encrypted filesystems.
```

Comment:
Is there a reason why this is only recommended for an error log scrub? Don't keys need to be loaded anyway to get exact file-names when an error is found in a full scrub?

If there is some aspect specific to this type of scrub specifically I think it might be worth mentioning in the documentation, specifically, what is the consequence of **not** having the keys loaded? It being strongly advised doesn't really tell the user anything about why they _should_ load the keys.

And if there's a strong enough reason, might it be worth asking for confirmation before running an error log scrub with keys unloaded (with an override to just do it and/or option to load them)?

Apologies for my ignorance on the specifics, just thought it might be worth asking from the perspective of a likely future user of the feature. 😉